docs/literate/make.jl

```diff
@@ -0,0 +1,122 @@
+using Literate
+using Test
+using HTTP
```

Comment:
Where do you use this package? Which functions do you use? Could you please import them explicitly?

Reply:
I just use it to check the URLs via `HTTP.get(link, retry=false, connect_timeout=15)` at the end of `docs/make`.
So I just do `using HTTP: get`, right?

Anyway, maybe you have a better idea for this link checking part. I think this could be temporary at all.

Reply:
No, I don't have a better idea right now. Did you look at Documenter.jl? If I remember correctly, they also allow some link checking. Otherwise, you can keep this as it is and use my suggestion above to make it clear that everything from HTTP.jl will be prefixed accordingly.

Reply:
Then we can get rid of it from my point of view (in this file and the Project.toml requiring HTTP)

Reply:
I just think about if it's necessary to check the links to the notebooks in binder and nbviewer. I mean, if everything works, they are uploaded automatically and no mistakes should happen.
(Yes, there is a feature from Documenter to check URL. The problem were the notebook links, which were not working until the notebook was uploaded first. But that was never going to happen because of the error, the check creates.)

Anyway, if we don't (need to) check the notebook links, everything should work now.